FEMPlateInMembraneActionAnalysis/solutionMatricesAndVectors/computeStiffnessMatrixPlateInMembraneActionLinear.m

```diff
@@ -50,10 +50,10 @@
 for i=1:length(mesh.elements)
     
     % Get the current element in the mesh
-    element = mesh.elements(i,1:no_nodes_element);
+    element = mesh.elements(i,2:no_nodes_element);
```

Comment:
Yes, you are right. I missed that. This is what I meant by "a file that I missed" :) I will check the changes again

Reply:
Ahh great I spotted that :) The easier would be to do a search in all files within the cane/ directory for **.elements** and **.nodes**, which will give you all instances you will have to change. Under linux you can try the command,

grep -r "\.nodes"
grep -r "\.elements"

I will perform the command and send you the outcome in a separate e-mail so that you can double check that all corresponding instances have been found